packages/dotnet/ReadMe/HarJsonTranslationLogics/HarJsonBuilder.cs

```diff
@@ -1,60 +1,60 @@
-﻿using Microsoft.AspNetCore.Http;
-using Microsoft.Extensions.Configuration;
-using Newtonsoft.Json;
-using ReadMe.HarJsonObjectModels;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Runtime.InteropServices;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using ReadMe.HarJsonObjectModels;
 
 namespace ReadMe.HarJsonTranslationLogics
 {
   public class HarJsonBuilder
   {
-    private readonly RequestDelegate _next;
-    private readonly HttpContext _context;
-    private readonly DateTime _startDateTime;
-    private readonly IConfiguration _configuration;
-    ConfigValues _configValues;
+    private readonly RequestDelegate next;
+    private readonly HttpContext context;
+    private readonly DateTime startDateTime;
+    private readonly IConfiguration configuration;
+    ConfigValues configValues;
     string guid = null;
 
     public HarJsonBuilder(RequestDelegate next, HttpContext context, IConfiguration configuration, ConfigValues configValues)
     {
-      _next = next;
-      _context = context;
-      _startDateTime = DateTime.UtcNow;
-      _configuration = configuration;
-      _configValues = configValues;
+      this.next = next;
```

Comment:
Kind of wild that you can do either `_next` or `this._next` and it'll pick up the class property `_next`. That seems like it'd be some sort of JS `strict mode` equivalent in .NET but I'm unable to find anything online about it outside of the [SA1101](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1101.md) StyleCop rule that was warning about it.